arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -1028,8 +1028,9 @@
           "description": "The SKU name of the the container registry. Required for registry creation.",
           "enum": [
             "Basic",
-            "Standard",
-            "Premium"
+            "Standard_Small",
+            "Standard_Medium",
+            "Standard_Large"
```

Comment:
Does this mean the service has already changed? If so, how are you handling requests from existing code that will still come in with values, say "premium" ? I'm trying to understand if this is a breaking change or not.